bin/i18n/src/i18n/common.clj

```diff
@@ -36,7 +36,7 @@
     {:id                (.getMsgid message)
      :id-plural         (.getMsgidPlural message)
      :str               (.getMsgstr message)
-     :str-plural        (seq (remove str/blank? (.getMsgstrPlural message)))
+     :str-plural        (seq (.getMsgstrPlural message))
```

Comment:
@camsaul Can you help take a look at this? Was there any specific reason we're removing blank string from plurals?

Reply:
I don't know exactly _why_ we're doing it. These scripts are just a port of some old node.js scripts written by someone who left years ago... I just made the new scripts return exactly the same thing as the old ones. I don't know why the old ones did this, but if it doesn't make sense then we should change it